apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -415,3 +421,18 @@ registerPlugin( 'page-templates-sidebar', {
 		);
 	},
 } );
+
+// Make sidebar plugin open by default.
+const unsubscribe = subscribe( () => {
+	if (
+		! wpSelect( 'core/edit-post' ).isEditorPanelOpened(
+			'page-templates-sidebar/Template Modal Opener'
+		)
+	) {
+		wpDispatch( 'core/edit-post' ).toggleEditorPanelOpened(
+			'page-templates-sidebar/Template Modal Opener'
+		);
+
+		unsubscribe();
```

Comment:
One correction from my original comment. We should put `unsubscribe` at the end of the callback, not inside the if statement. Otherwise, if the editor panel was opened by default, if you click to close it, it won't close the first time because this is still listening!